tests/unit-core/test-api.c

```diff
@@ -1114,8 +1117,7 @@ main (void)
   res = jerry_eval (test_magic_str_access_src,
                     sizeof (test_magic_str_access_src) - 1,
                     JERRY_PARSE_NO_OPTS);
-  TEST_ASSERT (jerry_value_is_boolean (res));
-  TEST_ASSERT (jerry_get_boolean_value (res) == true);
+  TEST_ASSERT (jerry_value_is_true (res));
```

Comment:
Please don't remove existing asserts.

Reply:
Keep as it. That's the intention of the PR.

Reply:
> Keep as it. That's the intention of the PR.

There are many similar tests use jerry_value_is_boolean and jerry_get_boolean_value. I can't see why this should be removed and other should be kept. I can't see any problem with have more tests to test all of these API functions.